clusterloader2/pkg/measurement/common/slos/pod_startup_latency.go

```diff
@@ -274,14 +276,14 @@ func (p *podStartupLatencyMeasurement) gatherScheduleTimes(c clientset.Interface
 	return nil
 }
 
-func (p *podStartupLatencyMeasurement) checkPod(obj interface{}) {
+func (p *podStartupLatencyMeasurement) checkPod(_, obj interface{}) {
```

Comment:
Does this handle nil correctly?

Reply:
There is a obj cast to pod interface. IIRC casting nil always returns (nil, false).
Anyway, I've just added explicit nil check in method passed to informer.